voxpupuli-acceptance.gemspec

```
@@ -23,11 +23,11 @@ Gem::Specification.new do |s|
   s.add_dependency 'beaker_puppet_helpers', '~> 2.2'
   s.add_dependency 'beaker-rspec', '~> 8.0', '>= 8.0.1'
   s.add_dependency 'beaker-vagrant', '~> 1.2'
+  s.add_dependency 'puppet_fixtures', '~> 0.1'
```

Semver says you should pin it like this
```suggestion
  s.add_dependency 'puppet_fixtures', '~> 0.1.0'
```